change/@fluentui-react-components-25dd8a93-fd84-4b2a-a2bc-68b18e647d9d.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

```suggestion
  "type": "none",
```

This should be `none` as there is nothing in this PR that affects distributed code.